Kconfig.zephyr

```diff
@@ -843,6 +843,7 @@ endmenu
 
 config LEGACY_INCLUDE_PATH
 	bool "Allow for the legacy include paths (without the zephyr/ prefix)"
+	default y
```

Comment:
let's fix legacy includes instead (there's a script to automate the process in `scripts/utils`).

Reply:
It will be an ongoing task but due to to needing lots of changing in various components e.g. docker images, this will have to be changed back to n during the next upmerge process